pages/find/whoami.vue

```diff
@@ -10,18 +10,11 @@
           <b-col class="text-muted text-center">
             <p>We need your email address to let you know when you have replies.  We won't give your email to anyone else.</p>
             <p>You will get emails from us, which you can control or turn off from Settings.</p>
-            <b-form-input
-              ref="email"
-              v-model="email"
-              type="email"
-              size="lg"
-              class="d-inline-block form-control email"
-              placeholder="What's your email address?"
-            />
+            <EmailValidator :email.sync="email" :valid.sync="emailValid" center />
```

Comment:
On  this page I'm getting "aria-describedby="undefined-feedback".  And the label associcated with it has a "for" of "email-"

Reply:
I don't think you need the aria attribute as long as your label for is correct

Reply:
Thanks.  This comes from ValidatingFormInput, which Nick wrote.  It looks like it implicitly (inside validationFieldHelpers) expects an id, but doesn't force that to happen in any way, and there are examples where it doesn't.  This probably needs some unravelling, if you fancy it.

Reply:
I'll create an issue for it now so it doesn't get forgotten then.